olive/model/handler/mixin/json.py

```diff
@@ -15,14 +16,23 @@ class JsonMixin:
     Different model handler need to override the behavior to add its own attributes.
     """
 
+    # keys for config parameters that are not part of resource_keys or model_attributes
+    # self.{key} must be defined for each key in jsonify_config_keys and should be serializable
+    jsonify_config_keys: Tuple[str, ...] = ()
```

Comment:
jsonify seems a verb. we'd better use a noun? how about jsonity?

Reply:
changed to `json_config_keys`